packages/rest/src/__tests__/unit/rest.server/rest.server.unit.ts

```diff
@@ -201,6 +201,25 @@ describe('RestServer', () => {
         const expressApp = server.expressApp;
         expect(expressApp.get('strict routing')).to.equal(true);
       });
+      it('honors openApiSpec.info in config', async () => {
+        const spec = {
+          info: {
+            name: 'Test',
+            version: '1.0.0-beta.0',
+          },
+        };
+
+        const app = new Application({
+          rest: {
+            openApiSpec: {
+              schema: spec,
```

Comment:
The property name should NOT be `schema`. Let's use something like `initialObject`.

Reply:
Why not just merge the whole openApiSpec object?

Reply:
> Why not just merge the whole openApiSpec object?

Not the whole object is the schema openapi

Reply:
> The property name should NOT be `schema`. Let's use something like `initialObject` or `baseDocument`.

Solved